meta-ivi-test/recipes-extended/persistence-client-library/persistence-client-library/0001-memset-improperly-initialized-length-of-elements.patch

```diff
@@ -0,0 +1,309 @@
+diff --git a/test/persistence_client_library_test_file.c b/test/persistence_client_library_test_file.c
```

Comment:
Me no like... ;-) Why don't we upstream the patch instead?  
(By the way, I have commit access to these repos, just in case the maintainer is not responsive, if that's the issue) and this change seems clear enough.